docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -242,33 +242,35 @@ The following image shows a line chart with the **Green-Yellow-Red (by value)**
 
 #### Line style
 
-Set the style of the line. To change the color, use the standard [color scheme](ref:color-scheme) field option.
+Choose a solid, dashed, or dotted line style:
 
-- **Solid:** Display a solid line. This is the default setting.
-- **Dash:** Display a dashed line. When you choose this option, a list appears for you to select the length and gap (length, gap) for the line dashes. Dash spacing set to 10, 10 (default).
-- **Dots:** Display dotted lines. When you choose this option, a list appears for you to select the gap (length = 0, gap) for the dot spacing. Dot spacing set to 0, 10 (default)
+- **Solid** - Display a solid line. This is the default setting.
+- **Dash** - Display a dashed line. When you choose this option, a list appears for you to select the length and gap (length, gap) for the line dashes. Dash spacing is 10, 10 by default.
+- **Dots** - Display dotted lines. When you choose this option, a list appears for you to select the gap (length = 0, gap) for the dot spacing. Dot spacing is 0, 10 by default.
 
 ![Line styles examples](/static/img/docs/time-series-panel/line-styles-examples-v9.png)
 
 {{< docs/shared lookup="visualizations/connect-null-values.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}
 
 {{< docs/shared lookup="visualizations/disconnect-values.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}
 
+To change the color, use the standard [color scheme](ref:color-scheme) field option.
+
 #### Show points
 
-You can configure your visualization to add points to lines or bars.
+Set whether to show data points to lines or bars. Choose from the following:
 
-- **Auto:** Grafana determines to show or not to show points based on the density of the data. If the density is low, then points appear.
-- **Always:** Show the points regardless of how dense the data set is.
-- **Never:** Do not show points.
+- **Auto** - Grafana determines whether to show or not to show points based on the density of the data. If the density is low, then points appear.
+- **Always** - Show the points regardless of how dense the data set is.
+- **Never** - Don't show points.
 
 #### Stack series
 
-_Stacking_ allows Grafana to display series on top of each other. Be cautious when using stacking in the visualization as it can easily create misleading graphs. To read more about why stacking might not be the best approach, refer to [The issue with stacking](https://www.data-to-viz.com/caveat/stacking.html).
+Set whether Grafana stacks or displays series on top of each other. Be cautious when using stacking in the visualization because it can easily create misleading graphs. To read more about why stacking might not be the best approach, refer to [The issue with stacking](https://www.data-to-viz.com/caveat/stacking.html). Choose from the following:
```

Comment:
```suggestion
Set whether Grafana stacks or displays series on top of each other. Be cautious when using stacking as it can create misleading graphs. To read more about why stacking might not be the best approach, refer to [The issue with stacking](https://www.data-to-viz.com/caveat/stacking.html). Choose from the following:
```

Reply:
Just changed one part of upir suggestion since "because" is clearer than "as" but made the update otherwise.